tests/unit/webhookTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+class webhookTest extends PHPUnit_Framework_TestCase
+{
+    protected function setUp()
+    {
+        $this->pusher = new Pusher\Pusher('thisisaauthkey', 'thisisasecret', 1, true);
+        $this->auth_key = 'thisisaauthkey';
+    }
+
+    public function testValidWebhookSignature()
+    {
+        $signature = '40e0ad3b9aa49529322879e84de1aaaf18bde1efe839ca263d540cc865510d25';
+        $body = '{"hello":"world"}';
+        file_put_contents('php://stdout', $body);
+        $headers = array(
+            'X-Pusher-Key'       => $this->auth_key,
+            'X-Pusher-Signature' => $signature,
+        );
+        file_put_contents('php://stdout', "\n Bonobo\n");
```

Comment:
I'm assuming these were left in by accident?

Reply:
Correct, whoops!